README.md

```diff
@@ -39,6 +39,12 @@ To determine Telegram room parameters, follow the procedure:
    ```
 5. To make all room messages visible to the bot, make it a room admin
 
+### Build
+
+```sh
```

Comment:
Use `console` instead of `sh`. That section is not a proper shell script, but a console transcript; there's difference.